### PR TITLE
some changes to generate same input for c and js

### PR DESCRIPTION
**Pull request by naturegift**
Since in Javascript, bitwise operations are performed on 32 bit integers, some longs have been changed to ints in the C code to make them equivalent. Also, since Javascript uses 32 bit signed integers, some changes have been done in ziggurat to make them work in the same way.

### CODE REVIEW

**Comment by naturegift** [comment]
Hi Erick,

I was travelling back to Montreal, so I couldn't reply.
Yes, I checked the inputs and outputs when I  made changes.
I am back to Montreal today. But I got cough and cold, so I will check again as soon as I am able to get back to work.

Thanks & Regards,
Prabhjot

**Comment by elavoie** [comment]
No worries I also got a cold coming back from Europe...

File: implementations/js/spmv.js [hunk 1]
```diff
@@ -272,8 +274,15 @@ function runner (dim, density, stddev, iterations) {
     for(var i = 0; i < iterations; ++i) spmv_csr(m.Ax, dim, m.Arow, m.Acol, v, y, out);
     var t2 = performance.now();
 
-    console.log("The total time for the spmv is " + (t2-t1)/1000 + " seconds");
     console.log(JSON.stringify({ status: 1,
              options: "run(" + [dim, density, stddev].join(",") + ")",
-             time: (t2 - t1) / 1000 }))
+             time: (t2 - t1) / 1000,
+             output: {
+             row_ptr: parseInt(parseInt(fletcher_sum_ndarray(ndarray(m.Arow)))),
```
Comment (elavoie) [inline implementations/js/spmv.js]: Remove parseInts. All of them.

File: implementations/js/spmv.js [hunk 2]
```diff
@@ -272,8 +274,15 @@ function runner (dim, density, stddev, iterations) {
...
+             col: parseInt(parseInt(fletcher_sum_ndarray(ndarray(m.Acol)))),
+             val: parseInt(parseInt(fletcher_sum_ndarray(ndarray(m.Ax)))),
```
Comment (elavoie) [inline implementations/js/spmv.js]: same thing

**Pull request merged**